classes/person.rb

```diff
@@ -0,0 +1,29 @@
+class Person
+  attr_accessor :id, :name, :age, :parent_permession
```

Comment:
- [ ] The id should have only a reader. Ids are not really something we want to be writable once created.

  ```
  attr_reader :id
  ```